examples/podinfo/cue.mod/pkg/timoni.sh/core/v1alpha1/image.cue

```diff
@@ -0,0 +1,19 @@
+package v1alpha1
+
+// Image defines the schema for an OCI image reference.
+#Image: {
+	repository: string
+	tag:        string
+	digest:     string
```

Comment:
Let's add validation to digest value. 

The syntax is like this:
```
digest : string 
digest: =~ "<regex>"
```
Also, shouldn't it be optional or you prefer to avoid `!= _|_`?